__tests__/generate.spec.ts

```diff
@@ -163,15 +179,23 @@ describe('Generate', () => {
       # Rule extracted from dir8/package.json
       dir8/ gbuilder@builder.com other.friend@domain.com
```

Comment:
this line should change as well because it has the same treatment as having all the files inside owned by those users. (that's the intention of that line)

Reply:
we assume that all the files under that package should be covered by those owners so that one should be `/dir8/`?

Reply:
I think we agreed to keep the leading slash out of this PR. So far this PR has only touched the patterns created from nested CODEOWNERS files, but I can update the patterns created from the package.json files also. 

If we want to keep the pattern from matching other directories of that name, it would need to be `dir8/**/* gbuilder@builder.com other.friend@domain.com` given we leave the leading slash out of this PR.